server/views/index.pug

```diff
@@ -27,7 +27,8 @@ html
             tracesTimePresetOptions: !{JSON.stringify(tracesTimePresetOptions || ['5m', '15m', '1h', '4h', '12h', '24h', '3d'])},
             timeWindowPresetOptions: !{JSON.stringify(timeWindowPresetOptions || [{shortName: '5m', longName: '5 minutes', value: 5 * 60 * 1000}, {shortName: '15m', longName: '15 minutes', value: 15 * 60 * 1000}, {shortName: '1h', longName: '1 hour', value: 60 * 60 * 1000}, {shortName: '6h', longName: '6 hours', value: 6 * 60 * 60 * 1000}, {shortName: '12h', longName: '12 hours', value: 12 * 60 * 60 * 1000}, {shortName: '24h', longName: '24 hours', value: 24 * 60 * 60 * 1000}, {shortName: '3d', longName: '3 days', value: 3 * 24 * 60 * 60 * 1000}, {shortName: '7d', longName: '7 days', value: 7 * 24 * 60 * 60 * 1000}, {shortName: '30d', longName: '30 days', value: 30 * 24 * 60 * 60 * 1000}])},
             tracesTTL: !{JSON.stringify(tracesTTL || -1)},
-            trendsTTL: !{JSON.stringify(trendsTTL || -1)}
+            trendsTTL: !{JSON.stringify(trendsTTL || -1)},
+            relatedTracesOptions: !{JSON.stringify(relatedTracesOptions || [])}
```

Comment:
Passing the relatedTraces configuration into the `haystackUiConfig` in the `windows` object.